projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -152,6 +157,46 @@ public function get_task( $id ) {
 		return $this->task_registry[ $id ];
 	}
 
+	/**
+	 * Get the required task ids for a given task list.
+	 *
+	 * @param array $task_list_id Task list.
+	 * @return array|null Required task ids.
```

Comment:
Should this not return `array` in all cases?
```suggestion
	 * @return array Required task ids.
```